internal/pkg/cli/app_delete.go

```diff
@@ -130,39 +129,14 @@ func (o *deleteAppOpts) askAppName() error {
 	return nil
 }
 
-func (o *deleteAppOpts) validateAppName() error {
-	if o.AppName == "" {
-		return nil
-	}
-
-	appNames, err := o.retrieveLocalAppName()
-	if err != nil {
-		return err
-	}
-	exists := false
-	for _, appName := range appNames {
-		if o.AppName == appName {
-			exists = true
-		}
-	}
-	if !exists {
-		return fmt.Errorf("input app %s not found", o.AppName)
-	}
-
-	return nil
-}
-
-func (o *deleteAppOpts) retrieveLocalAppName() ([]string, error) {
-	localApps, err := o.workspaceService.Apps()
+func (o *deleteAppOpts) retrieveAppName() ([]string, error) {
```

Comment:
Make the function name plural? `retrieveAppNames()`

Reply:
addressed